intro-HPC/gent/macros.tex

```diff
@@ -3,20 +3,22 @@
 \macro{\university}{UGent}
 \macro{\association}{AUGent}
 % HPC definitions
-\macro{\hpc}{HPC-UGent}
+\macro{\hpc}{UGent-HPC}
```

Comment:
I guess this should be changed in correspondence with the other macro's then.
Antwerp has `UA-HPC`, Leuven has `HPC-KU Leuven/UHasselt`, Brussels has `VUB-HPC`

So I guess Antwerp an Brussels can explain the 'the'
`The High Performance Computer` does make sense.

Since I do not want to go around changing other sites stuff in this pull request the only way to make it look good and consistent with the current usage of the \hpc this seemde like the easiest fix.
I propose to leave it like this, so we have a text that is at least readable.
In a future change we can decide to go with eiter:

* `institute-HPC` in each institutes macro
* `HPC-institute` in each institutes macro
* `s/the \hpc/\hpc` and put `the institute-HPC` or `HPC-institute` in the institues macro's
I prefer the last choice, but can we do this in a separate pull request then?